recommender/recommender.py

```diff
@@ -770,11 +854,33 @@ def student_view(self, _context=None):
         )
         frag.add_css_url("//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/themes/smoothness/jquery-ui.css")
         frag.add_javascript_url("//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/jquery-ui.min.js")
+        frag.add_javascript_url('//cdnjs.cloudflare.com/ajax/libs/mustache.js/0.8.1/mustache.min.js')
+        frag.add_javascript_url('//cdnjs.cloudflare.com/ajax/libs/intro.js/0.5.0/intro.min.js')
         frag.add_css(self.resource_string("static/css/tooltipster.css"))
         frag.add_css(self.resource_string("static/css/recommender.css"))
+        frag.add_css_url("http://usablica.github.io/intro.js/introjs.css")
```

Comment:
Any particular reason you're pulling this from Github, rather than hosting it yourself? It's going to change without you expecting it, which could cause this XBlock to break.